cleanrepo/Program.cs

```diff
@@ -595,7 +595,18 @@ private static void ListOrphanedIncludes(string inputDirectory, Dictionary<strin
         DirectoryInfo? rootDirectory = null;
 
         // Get all files that could possibly link to the include files
-        List<FileInfo>? files = HelperMethods.GetAllReferencingFiles("*.md", inputDirectory, ref rootDirectory);
+        List<FileInfo>? mdFiles = HelperMethods.GetAllReferencingFiles("*.md", inputDirectory, ref rootDirectory);
+        List<FileInfo>? ymlFiles = HelperMethods.GetAllReferencingFiles("*.yml", inputDirectory, ref rootDirectory);
+
+        List<FileInfo>? files = null;
+        if (mdFiles is not null)
+            files = mdFiles;
+
+        if(ymlFiles is not null)
+            if (files is null)
+                files = ymlFiles;
+            else
+                files.AddRange(ymlFiles);
 
         if (files is null || rootDirectory is null)
             return;
```

Comment:
```suggestion
        List<FileInfo>? mdfiles = HelperMethods.GetAllReferencingFiles("*.md", inputDirectory, ref rootDirectory);
        List<FileInfo>? ymlfiles = HelperMethods.GetAllReferencingFiles("*.yml", inputDirectory, ref rootDirectory);
        
        if ((mdfiles is null && ymlfiles is null) || rootDirectory is null)
            return;

        List<FileInfo> allReferencingFiles = mdfiles ?? [];
        if (ymlfiles is not null)
            allReferencingFiles.AddRange(ymlfiles);
```

Reply:
This is slightly more succinct. What do you think? If you like it, you'll also need to rename the loop variables on (new) line 605 (`files` and `markdownFile`).

Reply:
I'll go do that.

Reply:
I updated the loop variable.